### DISCUSSION TITLE
[Feature Request] Minimize to system tray

### DISCUSSION THREAD

**Discussion by lexcyn** [Chia-Network/chia-blockchain#8462] [Ideas]
When farming/plotting with the Chia GUI, instead of having the application open in the taskbar we should have the option of minimizing it to the system tray. Also it would be nice to have the option of "minimize to tray when closed" to avoid accidentally closing farming.

**Comment by warrengalyen** [comment]
I would really appreciate this feature as well. This is a really basic feature that I add to most of my Windows apps that run in the background. I really dislike having to keep the GUI open in the task bar.

**Comment by BrandtH22** [comment]
Hey @lexcyn , exciting news, the upcoming 2.3 release will have the ability to keep services running in the background when shutting down the GUI 🥳 

While this does not have an icon for the system tray it is a huge step towards that

**Comment by BrandtH22** [comment]
2.3 was just released with this running services in the background feature:

You can download 2.3 from here: https://www.chia.net/downloads/
This overviews how to use the background services feature: https://docs.chia.net/using-the-gui/#run-chia-services-in-the-background

Let us know if any questions arise👍

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
